inflammation-analysis.py

```diff
@@ -21,7 +21,7 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```

Comment:
This line is too long, split over multiple rows for readability.
the **-syntax is confusing and inconsistent, I would prefer explicit names. These changes depend on changes in models.py

```suggestion
        view_data = {
            'average': models.daily_mean(inflammation_data), 
            'max': models.daily_max(inflammation_data), 
            'min': models.daily_min(inflammation_data), 
            'stddev': models.daily_standard_deviation(inflammation_data))
        }
```